Formula/libpeas.rb

```diff
@@ -10,18 +10,43 @@ class Libpeas < Formula
     sha256 "0f521913ca0eaf13b55aacb75e4b87a730be1f527215741ae2ba207caac523b2" => :el_capitan
   end
 
+  option "with-python", "install support for running Python plugins"
```

Comment:
You've made it an option here but the default below. Can only be one or the other 😄.